pstoremgr/pstoremgr.go

```diff
@@ -123,11 +119,12 @@ func (pm *Manager) filteredPeerAddrs(p peer.ID) []api.Multiaddr {
 	return peerAddrs
 }
 
-// PeersAddresses returns the list of multiaddresses (encapsulating the
-// /ipfs/<peerID> part) for the given set of peers. For peers for which
-// we know DNS multiaddresses, we only return those. Otherwise, we return
-// all the multiaddresses known for that peer.
-func (pm *Manager) PeersAddresses(peers []peer.ID) []api.Multiaddr {
+// PeerInfos returns a slice of peerinfos for the given set of peers in order
+// of priority. For peers for which we know DNS
+// multiaddresses, we only include those. Otherwise, the PeerInfo includes all
+// the multiaddresses known for that peer. Peers without addresses are not
+// included.
+func (pm *Manager) PeerInfos(peers []peer.ID) []peerstore.PeerInfo {
```

Comment:
Easier to work with PeerInfos so changing this method. Note that returned slice is ordered by priority.